controllers/precache.go

```
@@ -313,6 +324,46 @@ func (r *ClusterGroupUpgradeReconciler) getPrecacheSpecTemplateData(
+			"kind":       "ManagedClusterAction",
+			"metadata": map[string]interface{}{
+				"name":      data.ResourceName,
+				"namespace": data.Cluster,
+			},
+			"spec": map[string]interface{}{
+				"actionType": "Create",
+				"kube": map[string]interface{}{
+					"resource": "configmap",
+					"template": map[string]interface{}{
+						"apiVersion": "v1",
+						"kind":       "ConfigMap",
```

golangci-lint (2.12.2)

[error] string `ConfigMap` has 3 occurrences, make it a constant

(goconst)